django/contrib/staticfiles/storage.py

```diff
@@ -151,7 +151,24 @@ def url(self, name, force=False):
         """
         return self._url(self.stored_name, name, force)
 
-    def url_converter(self, name, hashed_files, template=None):
+    def get_comment_blocks(self, content):
+        """
+        Return a list of (start, end) tuples for each comment block.
+        """
+        return [
+            (match.start(), match.end())
+            for match in re.finditer(r"\/\*.*?\*\/", content, flags=re.DOTALL)
```

Comment:
I think we should use a bit more complicated regular expression `\/\*[^*]*\*+([^/*][^*]*\*+)*\/` (see ["lexical scanner"](https://www.w3.org/TR/CSS22/grammar.html#scanner) and ["consume comments"](https://drafts.csswg.org/css-syntax-3/#consume-comment)).

Reply:
... also we should take into account only comments that contain `url` or `import`.